src/components/Settings/integrations/Mailchimp/MailchimpAccordion.tsx

```diff
@@ -337,7 +337,11 @@ export const MailchimpAccordion: React.FC<AccordionProps> = ({
         !showSettings && (
           <Box>
             <Alert severity="success" style={{ marginBottom: '20px' }}>
-              {t('Your contacts are now automatically syncing with MailChimp')}
+              {t(
+                `Your contacts are now automatically syncing with MailChimp. Changes 
```

Comment:
While you're at it, can you fix the capitalization? From their website it looks like "Mailchimp" is the correct way to capitalize this.

```suggestion
                `Your contacts are now automatically syncing with Mailchimp. Changes 
```